source/_components/hue.markdown

```diff
@@ -29,15 +29,21 @@ If you want to enable the component without relying on the [discovery component]
 hue:
   bridges:
     - host: DEVICE_IP_ADDRESS
+      allow_unreachable: true
+      allow_hue_groups: false
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
I understand that, but in this case it's very misleading, as at least I was assuming that:
```
hue:
  bridges:
    - host: 192.168.10.5
    - allow_unreachable: true
    - allow_hue_groups: true
```
was a valid and senseful configuration (due to all parameters being optional, this was valid before 0.66, but didn't make any sense).
So I wanted to give an example that shows the omitted dashes for additional parameters belonging to the same bridge.